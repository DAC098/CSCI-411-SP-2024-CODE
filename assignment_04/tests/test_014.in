4
-3 2 1
cgccccacccctccca gcagtagca
omxckwvigajjbbufprgsctvaip ezllzqoacrvxgqsauxacrjyqujaoyfemahhsq
tgctg gtgcgccta
cobzepqwhqxkhxmzm trlefqundbrqztffezunptvtnzehn
