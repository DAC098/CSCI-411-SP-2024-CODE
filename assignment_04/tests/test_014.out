_________cgccccacccctccca
gcagtagca________________
5
_____________________________________omxckwvigajjbbufprgsctvaip
ezllzqoacrvxgqsauxacrjyqujaoyfemahhsq__________________________
-59
_________tgctg
gtgcgccta_____
-17
_____________________________cobzepqwhqxkhxmzm
trlefqundbrqztffezunptvtnzehn_________________
-53
